package.json

```diff
@@ -3,7 +3,7 @@
   "engines": {
     "node": ">=6.9.5"
   },
-  "version": "7.0.0",
```

Comment:
personally no fond of bumping these versions along with the packaged marketing version, since the project didn't change APIs nor protocol.